.buildkite/pipeline.yaml

```diff
@@ -63,14 +63,6 @@ steps:
     <<: *elastic
   - wait: ~
 
-# Test Sorbet Gem Phase - Its own phase because it relies on the gem build built
```

Comment:
why was this deleted?

Reply:
It's moved to `test-static-sanitized.sh`, in the same test invocation as `@ruby_2_4_3` and `@gems`.